### PR TITLE
Update dublin bus transport url

### PR DESCRIPTION
**Pull request by Emilv2**
**Description:**

**Pull request in [home-assistant](https://github.com/home-assistant/home-assistant) (if applicable):** home-assistant/home-assistant#24922
## Checklist:

- [x] Branch: `next` is for changes and new documentation that will go public with the next [home-assistant](https://github.com/home-assistant/home-assistant) release. Fixes, changes and adjustments for the current release should be created against `current`.
- [x] The documentation follows the [standards][standards].

[standards]: https://developers.home-assistant.io/docs/documentation_standards.html

<a href="https://gitpod.io/#https://github.com/home-assistant/home-assistant.io/pull/9763"></a>

### CODE REVIEW

**Review by frenck — APPROVED** [review]

**Comment by klaasnicolaas** [comment]
✅ Can be merged after merging the parent PR 👍

**Comment by frenck** [comment]
Closing this PR, since the parent PR has been closed.

**Pull request closed without merging**
